### DISCUSSION TITLE
Is possible add timestamp to streaming video by restreamer ?

### DISCUSSION THREAD

**Discussion by Ota93** [datarhei/restreamer#660] [Q&A]
Hi, is possible add information about time and date to streaming video by restreamer ? Unfortunately, I haven't found anything about it yet. Thanks for answer.

**Comment by ioppermann** [comment]
No, this is currently not available in Restreamer. This requires a certain filter that is able to add some data into each frame. A better and more complete filter configuration is already on the feature request list.
